.github/CONTRIBUTING.md

```diff
@@ -1,65 +1,56 @@
 NewPipe contribution guidelines
 ===============================
 
-PLEASE READ THESE GUIDELINES CAREFULLY BEFORE ANY CONTRIBUTION!
-
 ## Crash reporting
 
-Do not report crashes in the GitHub issue tracker. NewPipe has an automated crash report system that will ask you to
-send a report via e-mail when a crash occurs. This contains all the data we need for debugging, and allows you to even
-add a comment to it. You'll see exactly what is sent, the system is 100% transparent.
+Report crashes in the automated crash report system of NewPipe.
+That way you get all data needed for debugging included in your bugreport for GitHub.
```

Comment:
```suggestion
This way all the data needed for debugging is included in your bug report for GitHub.
```

Reply:
I don't think 'bug report' can be written as one word, but it doesn't really matter, so I'm marking this as resolved.

Reply:
It is a bit daft not to. https://source.android.com/setup/contribute/read-bug-reports is consistently "bugreport", apart from the headline… :)

Reply:
No it isn't. I see 11 matches for "bugreport" and 15 matches for "bug report". The documentation was likely written by different people, but ultimately, as I said, it doesn't matter.